src/components/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Navbar, Nav } from 'react-bootstrap';
+import './header.css';
+import logo from '../logo2.png';
+
+
+function Header({logoImg, width, height, heading, theme, headerLinks}) {
+
+    // options for theme: {dark/light}
+
+    return (
+        <div style={{marginBottom:'40px'}}>
```

Comment:
Don't use inline-styling, and unnecessary div. Instead of div you can create an auxiliary component.